task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -55,7 +55,8 @@ spec:
           SNYK_TOKEN="$(cat ${SNYK_TOKEN_PATH})"
           export SNYK_TOKEN
         else
-          note="Task $(context.task.name) skipped: If you wish to use the Snyk code SAST task, please create a secret named snyk-secret with the key "snyk_token" containing the Snyk token."
+          to_enable_snyk=[here](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/testing_applications/enable_snyk_check_for_a_product/)
```

Comment:
I think this change might be causing [an issue](https://github.com/enterprise-contract/ec-cli/pull/1359#issuecomment-1959060259) in the latest version of the task.

This needs to be quoted with single quotes:
```suggestion
          to_enable_snyk='[here](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/testing_applications/enable_snyk_check_for_a_product/)'
```

Reply:
Thanks for noticing. I will fix it

Reply:
[PR](https://github.com/redhat-appstudio/build-definitions/pull/836) created for the fix